entities/boardTile.ts

```diff
@@ -0,0 +1,25 @@
+import { type } from "node:os";
+import { Column, Entity, JoinColumn, ManyToMany, ManyToOne, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+import Language from "./language";
+import User from "./user";
+
+@Entity()
+class BoardTile {
```

Comment:
This looks very similar to the PR https://github.com/Saphs/BrettSpiel_Backend/pull/33 please merge one and fix up the commits to clearly show the actual changes in this commit. Otherwise it makes the reviewing process confusing.